cime_config/testdefs/testlist_clm.xml

```diff
@@ -3332,7 +3332,7 @@
       <option name="tput_tolerance">0.1</option>
     </options>
   </test>
-  <test name="PFS_Ld10_PS" grid="f19_g17" compset="I2000Clm50BgcCrop" testmods="clm/matrixcnOn">
+  <test name="PFS_Ld10_PS" grid="f19_g17" compset="I2000Clm50BgcCrop" testmods="clm/default--clm/matrixcnOn">
```

Comment:
This was the only test with `matrixcnOn*` that didn't have another testmod as well.

Reply:
Are you adding visibility to what's already happening in `include_user_mods` or is there other significance to introducing "default" explicitly here?

Reply:
Oh, now I see that you removed "default" from the include_user_mods.